pull-request/README.md

```diff
@@ -0,0 +1,38 @@
+# Opening a Pull Request(PR)
+
+Think in your PR as something you are selling to your fellow developers, you the author are the seller, and the reviewers are the customers. You want to be as effective as possible in demostrating the value of your code, and why it is needed. This guide will try to help you accomplish that.
+
+1. Define scope (What are you tackling on this PR?):
+      
+      **User Stories/Features:**
+      
+      The maximum scope of a PR should be a complete User Story.
+            However, sometimes a single User Story may result in too many changes in the code base making your PR too         difficult to review and once reviewed, too difficult to address all the comments you received.  
```

Comment:
Completely agree. Would it make sense to suggest a maximum amount of lines changed? I believe 400 is a good number (excluding yarn.lock and that kind of stuff). Of course there can be exceptions but in general it happens to me that if a PR is bigger than that my code review quality goes down because I don't put the same attention to each line as if I were reviewing a small PR. What are your thoughts on this?

Reply:
Agreed! We already have danger check the amount of lines changed in some repos but suggesting it in this guides makes total sense, will update!